src/components/navigation-header/components/dropdown-menu/dropdown-menu.module.css

```diff
@@ -63,9 +63,13 @@
 	border-bottom-left-radius: 6px;
 	border-bottom-right-radius: 6px;
 	box-shadow: var(--token-elevation-higher-box-shadow);
-	min-width: 252px;
+	min-width: max-content;
+	position: fixed;
```

Comment:
Is it possible to change this back to `absolute` from`fixed`? Asking because having it as `fixed` has caused visual regressions. If you scroll down the page and then open the new menu on desktop on the homepage, it is not at the top of the page as expected. Same for other pages

<img width="1364" alt="Screenshot 2024-09-09 at 16 16 04" src="https://github.com/user-attachments/assets/c7bd0eff-3b3f-4011-95a1-e05b7900e7d3">


<img width="1222" alt="Screenshot 2024-09-09 at 16 14 42" src="https://github.com/user-attachments/assets/fc6515fd-9edd-40c1-8b11-ff51d9e25464">

Reply:
Yikes! Good call. Addressed here:
ec41196

I had gone for `fixed` initially for a less-bespoke `left` value to get the panel aligned. Is there anything you would suggest other than the hardcoded `left: -216px` I now have in place?